modules/subnet_router/efs.tf

```diff
@@ -22,7 +22,7 @@ resource "aws_efs_file_system" "tailscale" {
   }
 
   tags = {
-    Name = "${var.vpc}-tailscale"
+    Name = local.name
```

Comment:
Man I didn't realize how many places I had `"${var.vpc}-tailscale"` scattered, should've been a `local` to begin with!